bin/chrome2calltree.js

```diff
@@ -88,7 +88,8 @@ var launchKCacheGrind = function(logpath) {
 };
 
 readEntireStream(inStream, function(contents) {
-    chrome2calltree.chromeProfileToCallgrind(JSON.parse(contents), outStream);
+    var copy = false; // default to not mutate for efficiency
```

Comment:
The comment says this does the opposite of what it actually does :)

This is also not a default, since it isn't over-rideable here.

You can change this to:

```
var copy = false; // For efficiency, we mutate the profile in place
```